tcms/testruns/tests/test_permissions.py

```diff
@@ -128,3 +128,37 @@ def verify_post_with_permission(self):
             self.assertEqual(case.history.latest().history_id, case_run.case_text_version)
             self.assertEqual(last_run.build, case_run.build)
             self.assertEqual(None, case_run.close_date)
+
+
+class MenuAddCommentItemTestCase(PermissionsTestCase):
+    permission_label = 'django_comments.add_comment'
+    http_method_names = ['get']
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.test_run = factories.TestRunFactory()
+
+        cls.url = reverse('testruns-get', args=[cls.test_run.pk])
+        cls.post_data={'summary': 'mock',
+                                }
+        super().setUpTestData()
+
+        cls.add_comment_html = \
+            '<a href="#" class="addBlue9 js-show-commentdialog">{0}</a>' \
+            .format(_('Add'))
+
+    def assert_on_testrun_page(self, response):
+        self.assertContains(response, 'summary')
+        self.assertContains(response, 'Product')
```

Comment:
These 2 asserts are kind of pointless. They verify that a hard-coded value is present on the page, not that I am on the page which I want. Also these hard-coded strings are actually translated and will break your test.

Instead of asserting that "Product" is shown assert that the actual product name from the test run and the actual test run summary are shown. This will make it certain that the page you are looking at is the page displaying the test run object under test.